tests/testthat/test-shinytest2-tm_a_regression.R

```diff
@@ -0,0 +1,64 @@
+testthat::test_that("e2e - tm_a_regerssion: verify encoding values and widgets", {
+  skip_if_too_deep(5)
+
+  app <- app_driver_tm_a_regression()
+  app$expect_no_shiny_error()
+
+  testthat::expect_equal(
+    app$get_text("#teal-main_ui-root-active_tab > li.active > a"),
+    "Regression"
+  )
+
+  # Check MAIN encoding panel
+  # DO WE NEED any function for checking top of the encoding panel part?
+  encoding_dataset <- app$get_text("#teal-main_ui-root-regression .help-block")
+  testthat::expect_match(encoding_dataset, "Dataset:[\n ]*CO2", all = FALSE)
+
+
+  # Check plot settings
+  testthat::expect_identical(
+    app$get_active_module_input("plot_type"),
+    "Normal Q-Q"
+  )
+
+  testthat::expect_identical(
+    app$get_active_module_input("regressor-dataset_CO2_singleextract-select"),
+    "conc"
+  )
+  app$set_module_input("regressor-dataset_CO2_singleextract-select", "Treatment")
```

Comment:
Comment: method should be named as `set_active_module_input` while getter is named `get_active_module_input`

Reply:
Duly noted! Prepared a PR in `{teal}` for that https://github.com/insightsengineering/teal/pull/1204